pkgs/development/tools/godot/3/mono/glue.nix

```
@@ -0,0 +1,33 @@
+{ godot, mono }:
+
+godot.overrideAttrs (self: base: {
```

```suggestion
godot.overrideAttrs (oldAttrs: {
```
This would be the standard naming

---

I just grepped the repository, and I see `self` and `super` used in a lot of places.  It doesn't appear there is really a standard convention for this.

---

overrideAttrs should only get a single argument, while you are using two. I think it is falling back to constructing an override which is not what we want here.

---

Per the nixpkgs manual, a two-parameter lambda is also supported for overrideAttrs:  <https://nixos.org/manual/nixpkgs/unstable/#sec-pkg-overrideAttrs>.

It also works based on testing.